x-pack/plugins/lens/public/editor_frame_service/editor_frame/state_helpers.ts

```diff
@@ -37,6 +40,16 @@ import { readFromStorage } from '../../settings_storage';
 import { loadIndexPatternRefs, loadIndexPatterns } from '../../data_views_service/loader';
 import { getDatasourceLayers } from '../../state_management/utils';
 
+const COLOR_MAPPING: SuggestionRequest['mainPalette'] = COLOR_MAPPING_OFF_BY_DEFAULT
```

Comment:
I don't want to use the word legacy because we haven't concluded what we want to do. I think the problem is not the COLOR_MAPPING_OFF_BY_DEFAULT variable because this is correct. It sets the color mapping off. I think the problem is the COLOR_MAPPING. I could rename it to COLORING_METHOD which makes more sense.